src/cascadia/TerminalApp/CommandPalette.xaml

```diff
@@ -34,221 +34,201 @@ the MIT License. See LICENSE in the project root for license information. -->
-                            <ColumnDefinition Width="Auto"/>
-                            <!-- command label -->
-                            <ColumnDefinition Width="*"/>
-                            <!-- key chord -->
-                            <ColumnDefinition Width="16"/>
-                            <!-- gutter for scrollbar -->
-                        </Grid.ColumnDefinitions>
-
-                        <IconSourceElement
-                                    Grid.Column="0"
-                                    Width="16"
-                                    Height="16"
-                                    IconSource="{x:Bind Item.Icon,
-                                                        Mode=OneWay,
-                                                        Converter={StaticResource IconSourceConverter}}"/>
-
-                        <local:HighlightedTextControl
-                                    Grid.Column="1"
-                                    HorizontalAlignment="Left"
-                                    Text="{x:Bind HighlightedName, Mode=OneWay}"/>
-
-                        <!-- The block for the key chord is only visible
-                                when there's actual text set as the label. See
-                                CommandKeyChordVisibilityConverter for details.
-                             We're setting the accessibility view on the
-                                border and text block to Raw because otherwise,
-                                Narrator will read out the key chord. Problem is,
-                                it already did that because it was the list item's
-                                "AcceleratorKey". It's redundant. -->
-                        <Border
-                                    Grid.Column="2"
-                                    Visibility="{x:Bind Item.KeyChordText,
-                                                 Mode=OneWay,
-                                                 Converter={StaticResource CommandKeyChordVisibilityConverter}}"
-                                    Style="{ThemeResource KeyChordBorderStyle}"
-                                    Padding="2,0,2,0"
-                                    HorizontalAlignment="Right"
-                                    VerticalAlignment="Center"
-                                    AutomationProperties.AccessibilityView="Raw">
-
-                            <TextBlock
-                                        Style="{ThemeResource KeyChordTextBlockStyle}"
-                                        FontSize="12"
-                                        Text="{x:Bind Item.KeyChordText, Mode=OneWay}"
-                                        AutomationProperties.AccessibilityView="Raw" />
-                        </Border>
-
-                    </Grid>
-                </ListViewItem>
+                                  AutomationProperties.AcceleratorKey="{x:Bind Item.KeyChordText, Mode=OneWay}"/>
```

Comment:
I'm surprised this binding actually works! It's on the parent ListViewItem, but that ListViewItem is never bound to the actual object in our code. Is this just something that ListView handles for us?